### PR TITLE
:construction_worker: [maykinmedia/objects-api#463] Add trivy image scan

### PR DESCRIPTION
**Pull request by stevenbal**
Fixes maykinmedia/objects-api#463 partially

### CODE REVIEW

**Comment by stevenbal** [comment]
@annashamray I can't run the cache update job yet because it's not on main yet

File: .github/workflows/trivy.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Call Update Trivy Cache
```
Comment (annashamray) [inline .github/workflows/trivy.yml:1]: Sorry a little nitpicking. Correct me if I'm wrong but this workflow doesn't trigger trivy scan, it just cached the DB, right? 
So maybe let's rename it to trivy-db-cache of something like this?

File: .github/workflows/trivy.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+on:
+  workflow_dispatch: # Manual trigger for testing purposes
+  schedule:
+    - cron: '51 0 * * *'  # Run daily
+
+jobs:
+  call-update-trivy-cache:
+    uses: maykinmedia/open-api-workflows/.github/workflows/ci.yml@issue/fix-trivy
```
Comment (annashamray) [inline .github/workflows/trivy.yml:10]: Let's merge it in the upstream repo and then use it

**Review by annashamray — CHANGES_REQUESTED** [review]

File: .github/workflows/trivy.yml (hunk 1 above)
Comment (stevenbal) [reply .github/workflows/trivy.yml]: changed it

**Review by annashamray — CHANGES_REQUESTED** [review]

File: .github/workflows/trivy-db-cache.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: Call Update Trivy Cache
+
+on:
+  workflow_dispatch: # Manual trigger for testing purposes
+  schedule:
+    - cron: '51 0 * * *'  # Run daily
+
+jobs:
+  call-update-trivy-cache:
+    uses: maykinmedia/open-api-workflows/.github/workflows/ci.yml@v3
```
[the remark this answers is not in the document]
Comment (stevenbal) [reply .github/workflows/trivy-db-cache.yml:10]: It's part of `ci.yml` https://github.com/maykinmedia/open-api-workflows/blob/main/.github/workflows/ci.yml#L112

[the remark this answers is not in the document]
Comment (stevenbal) [reply .github/workflows/trivy-db-cache.yml]: nvm I'm blind

**Comment by stevenbal** [comment]
Turns out trivy has caching enabled by default now (https://github.com/aquasecurity/trivy-action/pull/399), so I'll close this branch and remove the unnecessary changes in objecttypes

**Review by annashamray — APPROVED** [review]

**Pull request merged**